easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -687,16 +687,18 @@ def sanity_check_step(self, *args, **kwargs):
             }
 
         # make sure 'exts_filter' is defined, which is used for sanity check
-        if self.multi_python:
-            # when installing for multiple Python versions, we must use 'python', not a full-path 'python' command!
-            if 'exts_filter' not in kwargs:
-                kwargs.update({'exts_filter': EXTS_FILTER_PYTHON_PACKAGES})
-        else:
-            # 'python' is replaced by full path to active 'python' command
+        exts_filter = EXTS_FILTER_PYTHON_PACKAGES
+
+        if self.toolchain.options.get('usempi', None):
+            # packages using MPI have to execute the sanity checks with mpirun
+            exts_filter = ("mpirun -n 1 " + exts_filter[0], exts_filter[1])
```

Comment:
This should either be configurable or should use `mpiexec` from the standard. `mpirun` would fail on the JSC systems where you are required to use `srun`.

Reply:
I always forget that `mpirun` is actually not part of the standard

Reply:
Changed it to `mpiexec` for simplicity